cmd/api/src/utils/test/auditlogs.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package test
```

Comment:
Thank you for moving this. I didn't like where it was, but also didn't want to spend time trying to find a better place to put it. This is a good place for it.

Reply:
Yep! I too wasn't sure where to put it so if you like the current directory, I like the current directory. 🙇